javascript/javascript1/week3/Note_taking/notes.js

```diff
@@ -0,0 +1,35 @@
+const notes = [];
+function addNote(content, id) {
+    notes.push({content:content,id:id});
+
+}
+addNote("First Note", 1);//added 1st note
+addNote("Second Note", 2);//added 2nd note
+
+
+function getNoteFromId(id){
+for(let i=0 ;i < notes.length; i++ ){
```

Comment:
you're not using `i` from the for loop at all. In fact you don't need a for loop because you're doing `findIndex` (somewhere inside of that there's probably a for loop)